projects/extension/sql/idempotent/015-load_dataset.sql

```diff
@@ -0,0 +1,43 @@
+create or replace function ai.load_dataset
```

Comment:
we sure we want this to be a function rather than a procedure with commits after each batch?